src/main/scala/mesosphere/marathon/api/akkahttp/v2/PodsController.scala

```diff
@@ -147,7 +149,23 @@ class PodsController(
       }
     }
 
-  def status(podId: PathId): Route = ???
+  @SuppressWarnings(Array("all")) // async/await
+  def status(podId: PathId): Route =
+    authenticated.apply { implicit identity =>
+      podManager.find(podId) match {
+        case None =>
+          reject(Rejections.EntityNotFound.noPod(podId))
+        case Some(pod) =>
+          authorized(ViewRunSpec, pod).apply {
+            onSuccess(podStatusService.selectPodStatus(podId)) { maybeStatus =>
+              // If selectPodStatus returns None this is a bug since find(podId) already verifies that the pod exists.
+              // We don't filter the pods with an authentication since we check for authorization before.
```

Comment:
authentication or authorization? I think you wanted to use word authorization in both cases in this sentence, right?

Reply:
It should be "authorization". Thanks.